### DISCUSSION TITLE
TFT35_E3_V2 Bootloader

### DISCUSSION THREAD

**Discussion by YoAdriaaannn** [bigtreetech/BIGTREETECH-TouchScreenFirmware#1575] [General]
I cannot update my firmware. It would seem updating the bootloader is the thing to do.

Does anyone know if the TFT35_V2 bootloader works on the E3 version of the display? If not, is there a bootloader available for the E3 board someone can point me to or post up?

Thanks!

**Comment by oldman4U** [comment]
There is such a TFT available. Did not know.

**Reply by YoAdriaaannn** [reply]
Indeed, there is such a display.
Made as a drop in for the Ender 3. The board differs where the rotary encoder knob is and the reset button. From what I can see, most of it is the same as the standard TFT35, but there are are differences in the component layout of the board.

https://www.bigtree-tech.com/products/bigtreetech-tft35-v3-0-touch-screen-display-marlin-and-tft-touch-mode-smart-controller-panel-for-skr-v1-4-skr-pro-skr-mini-board-of-3d-printer-part.html

Side note, my issues with not being able to update the firmware turned out to be an issue other than the bootloader.

**Reply by oldman4U** [reply]
This is v3 and not v2 like you described above. Good to hear the problem is solved. 

Please help the community and close the ticket once you do not need it anymore. Thank you

**Comment by oldman4U** [comment]
Somehow this reply went into the wrong thread. Sorry

Tft35 v2 Bootloader instruction is available.
